minio/minio
# Configuration of AMQP notification for exchange and publishing of event

## Description
Allow us to set publishing headers and exchange arguments

## Motivation and Context
Allows us to set the exchange arguments and message headers when publishing an event using AMQP.

## Regression
No

## How Has This Been Tested?
* Enable all events for a bucket
* Upload, update and delete a file
* Confirm that the event is in the queue
* Confirm that the exchange has to correct arguments when created
* Confirm that the message has to correct headers

## Types of changes

## Checklist:
